### ISSUE TITLE
Rounded Outline Icon only buttons are neither rounded nor outline

### ISSUE THREAD

**Issue by harleybl** [primefaces/primeng#9150]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

I took the code from the documentation here for rounded and outlined Icon buttons here:
https://www.primefaces.org/primeng/showcase/#/button

And pasted it into the stakblitz here:
https://stackblitz.com/edit/primeng-button-demo-rjmwrn

**Current behavior**
The bottom row of buttons are square and not outlined
![image](https://user-images.githubusercontent.com/14912198/89819322-38b0d600-db19-11ea-8519-1298f215e879.png)

**Expected behavior**
I expected the buttons to be rounded and outlined.
![image](https://user-images.githubusercontent.com/14912198/89819293-2afb5080-db19-11ea-86c1-d05137468918.png)

**Minimal reproduction of the problem with instructions**

**Please tell us about your environment:**
Windows, Angular 9, PrimeNG 9.1.2

* **Angular version:** 9.1.11

* **PrimeNG version:** 9.1.2

* **Browser:** Chrome
 
* **Language:** Typescript 3.8.3

**Comment by harleybl** [comment]
Partially it seems like the documentation is wrong on the site. I replaced the 'p-' prefix with 'ui-' on the class names.
The buttons are now rounded and showing the correct colors, but they are not appearing as outlined.

**Comment by Rotzbua** [comment]
Nope, the demo shows by default the latest version from github. Currently v10.
You have to select v9 to get the stable documentation.
